distribution/lib/Standard/Visualization/0.0.0-dev/src/Geo_Map.enso

```diff
@@ -28,5 +28,5 @@ json_from_table table =
    - value: the value to be visualized.
 process_to_json_text : Any -> Text
 process_to_json_text value = case value of
-    Table.Value _ -> json_from_table value . to_text
+    _ : Table -> json_from_table value . to_text
     _ -> value.to_json
```

Comment:
What happened here? Looking at this code I'd expect both the 'before' and 'after' to have basically the exactly same semantics, so I'm surprised this had to be changed to fix something (although I _do_ like the new code more, it is also prettier 🙂). But I'm wondering what was broken with the old code?

Reply:
I think something with the import changed. So `Table.Value` wasn't truly in scope. Switching to type match fixed it (and is cleaner).

Reply:
It is cleaner I agree.

I'm confused.

When the type `Table` to be matched is in scope, then it is also in scope for taking its member `Table.Value`. I can't imagine a scenario where one would work but not the other.

(Not trying to argue there was no error, but I'm trying to understand the situation and I'm confused)

Reply:
`Table` is brought in via the export within `Standard.Table`, not directly imported.
I agree it should be available so something odd.